src/Microsoft.Azure.WebJobs.Host/Executors/DefaultStorageCredentialsValidator.cs

```diff
@@ -55,6 +58,11 @@ internal class DefaultStorageCredentialsValidator : IStorageCredentialsValidator
                     account.Type = StorageAccountType.Premium;
                     return;
                 }
+                else if (isDevStoreAccount)
+                {
+                    // If using the storage emulator, it might not be running
+                    throw new InvalidOperationException(Constants.CheckAzureStorageEmulatorMessage);
```

Comment:
Please propagate the inner exception. i.e. pass the caught exception into the InvalidOperationException constructor

Reply:
Should we be doing this for the invalid credentials case as well?